server/src/completion.ts

```diff
@@ -76,9 +76,13 @@ export function initCompletionList(docs: Documentation, snippets: Snippets): voi
 
   predefinedCompletionListLight.push(
     ...Object.entries(snippets).map(([title, info]) => ({
-      label: title,
+      label: info.prefix,
       kind: CompletionItemKind.Snippet,
       data: { type: DataEntryType.Snippet },
+
+      detail: info.description,
+      insertText: info.body.join('\n'),
+      insertTextFormat: InsertTextFormat.Snippet,
```

Comment:
These 3 lines are unnecessary and added on CompletionResolve event with `enrichWithSnippetDetails` function.

Reply:
actually have to, otherwise those are not snip items.
those are working for auto completion, vs compared to completionResolve looks those for manual omni completion.